swagger/sdk_api.yaml

```diff
@@ -0,0 +1,142 @@
+                400:
+                    description: Bad Request
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/RestErrorMessage'
+    /sdk-init-project:
+        post:
+            summary: "Initialize SDK"
+            requestBody:
+                required: true
+                content:
+                    application/json:
+                        schema:
+                            $ref: '#components/schemas/SdkInitProjectRequest'
+            responses:
+                200:
+                    description: OK
+                400:
+                    description: Bad Request
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/RestErrorMessage'
+
+components:
+    schemas:
+        SdkInitSdkRequest:
+            properties:
+                data:
+                    $ref: '#/components/schemas/SdkInitSdkRequestData'
+                included:
+                    $ref: '#/components/schemas/SdkInitSdkRequestIncluded'
+
+        SdkInitSdkRequestData:
+            properties:
+                id:
+                    type: string
+                type:
+                    type: string
+                attributes:
+                    $ref: '#/components/schemas/SdkInitSdkRequestDataAttributes'
+                relationships:
+                    $ref: '#/components/schemas/SdkInitSdkRequestRelationships'
+
+        SdkInitSdkRequestDataAttributes:
+            properties:
+                developer_email:
+                    type: string
+                    example: 'developer@example.org'
+                developer_github_account:
+                    type: string
+                    example: 'Developer-Example.org'
+
+        SdkInitProjectRequest:
+            properties:
+                data:
+                    $ref: '#/components/schemas/SdkInitProjectRequestData'
+                included:
+                    $ref: '#/components/schemas/SdkInitProjectRequestIncluded'
+
+        SdkInitProjectRequestData:
+            properties:
+                id:
+                    type: string
+                type:
+                    type: string
+                attributes:
+                    $ref: '#/components/schemas/SdkInitSdkProjectDataAttributes'
+                relationships:
+                    $ref: '#/components/schemas/SdkInitProjectRequestRelationships'
+
+        SdkInitSdkProjectDataAttributes:
+            properties:
+                report_usage_statistics:
+                    type: boolean
+                    example: 'true'
+                    description: 'Do you agree to send anonymous usage reports to help improving the SDK?'
+                default_violation_output_format:
+                    type: string
+                    example: 'output'
```

Comment:
we don't have output here and this option

Reply:
This file removed